flatdata-generator/flatdata/generator/tree/nodes/trivial/enumeration.py

```diff
@@ -28,9 +31,19 @@ def create(properties, definition):
             result.insert(value_node)
             current_assigned_value += 1
 
-        bits_required = 0
+        # we do not want to genarate too many (exponential) values, so restrict to multiples of input size
+        if len(properties.enum_values) * 2 + 256 < 2 ** result.type.width:
+            # print("Props: " + str(properties.width) + " vs " + str(result.type.width))
```

Comment:
This print is not needed anymore, rigth?

Reply:
Removed